<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1355</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,99">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="help_info_button">
        <property name="text">
         <string>‚ùìÂú∫ÊôØËØ¥Êòé</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>ÊØèÊúàÂêå‰∏ö‰∫§ÊµÅÊï∞ÊçÆÊ±áÊÄªËÆ°ÁÆó</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="release_info_button">
        <property name="text">
         <string>üí°Âú∫ÊôØÊõ¥Êñ∞‰ø°ÊÅØ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_button">
        <property name="text">
         <string>ÈáçÁΩÆÊâÄÊúâ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="baoixan_code_rule_table">
          <property name="text">
           <string>ÈÖçÁΩÆÈô©Áßç‰ª£Á†ÅËßÑÂàô</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="baoxian_code_config_table">
          <row>
           <property name="text">
            <string>ÊÑèÂ§ñÈô©</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>ÂÅ•Â∫∑Èô©</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>ÂØøÈô©</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>ÂåªÁñóÂü∫Èáë</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Âπ¥ÈáëÈô©</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Èô©Áßç‰ª£Á†ÅÂºÄÂ§¥Ê†áËÆ∞</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Êñ∞Â¢ûÈô©Áßç‰ª£Á†Å</string>
           </property>
           <property name="background">
            <color>
             <red>207</red>
             <green>240</green>
             <blue>198</blue>
            </color>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ÂøΩÁï•Èô©Áßç‰ª£Á†Å</string>
           </property>
           <property name="background">
            <color>
             <red>242</red>
             <green>195</green>
             <blue>189</blue>
            </color>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>68</string>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>78</string>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
           </property>
          </item>
          <item row="1" column="2">
           <property name="text">
            <string>7824,7854</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>18</string>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string/>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
           </property>
          </item>
          <item row="3" column="1">
           <property name="text">
            <string>7824,7854</string>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>28</string>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
           </property>
          </item>
          <item row="4" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="upload_button">
            <property name="text">
             <string>1. ‰∏ä‰º†Êñá‰ª∂</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cal_button">
            <property name="text">
             <string>2. ÂºÄÂßãËÆ°ÁÆó</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="download_file_button">
            <property name="text">
             <string>3. ‰∏ãËΩΩÊñá‰ª∂</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="upload_info_text">
          <property name="text">
           <string>ÂΩìÂâçÂπ¥‰ªΩÔºö--ÔºåÊ±áÊÄªËÆ°ÁÆó --/--‰∏™ÊúàÂ∫¶Êï∞ÊçÆ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="upload_list"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
